azure-pipelines.yml

```diff
@@ -4,7 +4,7 @@ resources:
     - repository: templates
       type: github
       name: shotgunsoftware/tk-ci-tools
-      ref: refs/heads/master
+      ref: refs/heads/add-has_python_27-parameter
```

Comment:
To be reverted when https://github.com/shotgunsoftware/tk-ci-tools/pull/33 is merged